tools/accuracy/accuracy_checker.py

```diff
@@ -111,10 +119,40 @@ def get_np_datatype(in_type):
 def main():
     args = parse_args()
 
+    use_onnx = True
+    if args.onnx == None:
+        use_onnx = False
+    if not use_onnx and args.tf == None:
+        print('Error: please specify either an onnx or tf pb file')
+        sys.exit(-1)
+
     model_name = args.onnx
+
     batch = args.batch
 
-    model = migraphx.parse_onnx(model_name, default_dim_value=batch)
+    custom_inputs = args.input_dim
+
+    input_dims = {}
+    if custom_inputs != None:
+        for input in custom_inputs:
+            input_dim = ''.join(input.split(':')[:-1])
+            dims = [int(dim) for dim in input.split(':')[-1].split(',')]
+            input_dims[input_dim] = dims
+
+    if use_onnx:
+        if any(input_dims):
+            model = migraphx.parse_onnx(model_name,
+                                        default_dim_value=batch,
+                                        map_input_dims=input_dims)
```

Comment:
This is only for static shapes and doesn't work for dynamic shape

Reply:
okay, PR title mentions `dynamic input shape` so i thought that.  

by dynamic you mean users can specify batch size and input dims then ?

Reply:
updated PR title